src/common/transformations/src/transformations/common_optimizations/lora_subgraph_fusion.cpp

```diff
@@ -26,12 +27,19 @@ ov::pass::LoraSubgraphFusion::LoraSubgraphFusion() {
     auto lora_input_m = any_input();
     auto transpose_const1_m = wrap_type<ov::op::v0::Constant>(consumers_count(1));
     auto transpose1_m = optional<ov::op::v1::Transpose>({lora_input_m, transpose_const1_m}, consumers_count(1));
+
     auto read_value1_m = wrap_type<ov::op::util::ReadValueBase>();
-    auto matmul1_m = wrap_type<ov::op::v0::MatMul>({transpose1_m, read_value1_m}, consumers_count(1));
+    auto convert1_m = optional<ov::op::v0::Convert>(read_value1_m);
```

Comment:
probably it's need to check "consumers_count(1)" for these ops too

Reply:
Done